From c72489d4cdd041a26782a37ad0f6cd37d15f1c5e Mon Sep 17 00:00:00 2001
Message-ID: <c72489d4cdd041a26782a37ad0f6cd37d15f1c5e.1708471786.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1708471786.git.sharlatanus@gmail.com>
References: <cover.1708471786.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 28 Jan 2024 11:18:59 +0000
Subject: [PATCH 16/46] gnu: python-asdf-wcs-schemas: Update to 0.3.0.

* gnu/packages/astronomy.scm (python-asdf-wcs-schemas): Update to 0.3.0.
[propagated-inputs]: Remove python-asdf; add python-asdf-standard,
python-asdf-transform-schemas, and python-asdf-unit-schemas.
[native-inputs]: Add python-asdf and python-pytest-openfiles.

Change-Id: Iea9e49d7cd94d579c417d801aaff7a1c508c96b8
---
 gnu/packages/astronomy.scm | 17 +++++++++++++----
 1 file changed, 13 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 6630e60321..9583524660 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -4316,18 +4316,27 @@ (define-public python-asdf-astropy
 (define python-asdf-wcs-schemas
   (package
     (name "python-asdf-wcs-schemas")
-    (version "0.1.1")
+    (version "0.3.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "asdf_wcs_schemas" version))
        (sha256
-        (base32 "0khyab9mnf2lv755as8kwhk3lqqpd3f4291ny3b9yp3ik86fzhz1"))))
+        (base32 "1r9ls2skgwq6my68nfwfah1s0q6bm2qq97s0h79vwh997jy97rai"))))
     (build-system pyproject-build-system)
+    (arguments
+     (list
+      #:test-flags #~(list "tests")))
     (native-inputs
-     (list python-pytest python-setuptools-scm python-semantic-version))
+     (list python-asdf
+           python-pytest
+           python-pytest-openfiles
+           python-semantic-version
+           python-setuptools-scm))
     (propagated-inputs
-     (list python-asdf))
+     (list python-asdf-standard
+           python-asdf-transform-schemas
+           python-asdf-unit-schemas))
     (home-page "https://github.com/asdf-format/asdf-wcs-schemas")
     (synopsis "ASDF WCS Schemas")
     (description
-- 
2.41.0

